android/library/src/main/java/io/logto/client/config/LogtoConfig.kt

```diff
@@ -9,7 +11,8 @@ data class LogtoConfig(
 ) {
     val encodedScopes: String = scopes.joinToString(" ")
 
-    val cacheKey: String = "$clientId::$encodedScopes"
+    val cacheKey: String =
+        URLEncoder.encode("$clientId::$encodedScopes", "utf-8").replace("+", "%20")
```

Comment:
feels like we don't need to force align java encode `+` with js encode `%20`. Better keep the default encoding rule in each environment.

Reply:
Reasonable.

@wangsijie @xiaoyijun @gao-sun @darcyYe 

TBD: 

- Keep the default encoding rule of url-encoding in each environment? 
- Or stay consistency everywhere?

Reply:
They don't share data, no need to force align. The `replace` may cause confuse.

Reply:
Conclusion: Keep the default encoding rule of url-encoding in each environment.